### PR TITLE
enforce default audio and video bitrate

### PR DESCRIPTION
**Pull request by jpserra**
recent experiment showed that the [MediaRecorder](https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder/MediaRecorder) default video bitrate is not being respected. 

If we try the latest version of onfido-sdk on a recent iPhone, we can see very high bitrate (~10Mbps) which leads to very large videos being generated and causing the upload to fail

This MR explicitly sets the desired bitrate as an attempt to solve the identified issue

### CODE REVIEW

File: src/video.js [hunk 1]
```diff
@@ -11,6 +11,16 @@ const handleStop = (event) => {
 };
 
 const videoOptions = () => {
+  // TODO, These values are the same the MediaRecorder should have by default.
+  // We identified that in recent Safari versions the defaults were not being respected
```
Comment (kopijunkie) [inline src/video.js:15]: Nitpick: Would prefer the version(s) that this behaviour was found in is mentioned. Just so we know in future.

File: src/video.js [hunk 2]
```diff
@@ -11,6 +11,16 @@ const handleStop = (event) => {
...
+  // thus we have to add them here to ensure we controll the bitrate and keep file sizes resonable.
```
Comment (kopijunkie) [inline src/video.js:16]: Some minor typos here.
```suggestion
  // thus we have to add them here to ensure we control the bitrate and keep file sizes reasonable.
```

**Review by kopijunkie — COMMENTED** [review]
Just a couple of minor things. Otherwise looks good.

File: src/video.js (hunk 1 above)
Comment (jpserra) [reply src/video.js:15]: honestly don't know if we can really pinpoint a specific safari version - I'm not sure if this was an issue introduced by a specific safari version or is just a problem that gets exacerbated as more powerful iphones come out and are capable of recording at a higher bitrate

Comment (kopijunkie) [reply src/video.js:15]: Ah, ok. Guess we'll just leave the comment as it is then.

**Review by kopijunkie — APPROVED** [review]
LGTM

**Pull request merged**